src/client/src/apps/MainRoute/widgets/blotter/blotterTradesFilter.ts

```diff
@@ -1,56 +1,64 @@
 import { Trade } from 'rt-types'
-import { DEALT_CURRENCY, SYMBOL } from './blotterFields';
+import { DEALT_CURRENCY, SYMBOL } from './blotterFields'
 
-export type FieldValues = ReadonlyArray<any> | undefined;
+export type FieldValues = ReadonlyArray<any> | undefined
 
 export type BlotterFilters = {
   readonly [SYMBOL]?: FieldValues
   readonly [DEALT_CURRENCY]?: FieldValues
+  readonly count?: number
 }
 
-const tradeMatchesFilter = (trade: Trade, filterField: string, filteringFieldValues: FieldValues) => {
+const tradeMatchesFilter = (trade: Trade, filterField: string, filterValues: FieldValues) => {
   if (!trade) {
     return false
   }
-  if (!filteringFieldValues) {
+  if (!filterValues || filterValues.length === 0) {
     return true
   }
   if (!(trade as any).hasOwnProperty(filterField)) {
-    console.warn(`Trying to filter of field ${filterField} which does not exist in 'Trade' object`);
+    console.warn(`Trying to filter of field ${filterField} which does not exist in 'Trade' object`)
     return true
   }
 
   const tradeFieldValue = trade[filterField]
 
-  return filteringFieldValues.includes(tradeFieldValue)
+  return filterValues.includes(tradeFieldValue)
+}
+
+/**
+ * Fields by which we will filter trades
+ */
+const tradeFilterFields: ReadonlyArray<keyof Omit<BlotterFilters, 'count'>> = [
```

Comment:
complex type just to protect that values in array exist in `BlotterFilters` type